pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -328,8 +331,11 @@ filterAndCreateOverrides {
         prevAttrs.postPatch or ""
         + ''
           for path in $rmPatterns; do
-            rm -r "$path"
+            rm -r $path
```

Comment:
why get rid of double quotes? shouldn't they be included to prevent https://www.shellcheck.net/wiki/SC2086?